api/shape/centroid.js

```diff
@@ -0,0 +1,10 @@
+import Shape from './Shape.js';
+import { computeCentroid as op } from '@jsxcad/geometry';
+
+export const centroid = Shape.registerMethod3(
```

Comment:
Nice name 👍

Reply:
Yeah, I think it follows along with alignment better, given that this produces an oriented point at the centroid of the shape (rather than centering the shape).